### PR TITLE
explicit command to get tab completion install command

### PR DESCRIPTION
**Pull request by elisim**
## Motivation

(Write your motivation for proposed changes here.)

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebookresearch/hydra/blob/master/CONTRIBUTING.md)?

Yes/No

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

File: website/versioned_docs/version-1.0/tutorials/basic/running_your_app/6_tab_completion.md [hunk 1]
```diff
@@ -15,8 +15,8 @@ import Script from '@site/src/components/Script.jsx';
 
 
 ### Install tab completion
-Get the exact command to install the completion from `--hydra-help`.
+Get the exact command to install the completion from `python <your-hydra-main> --hydra-help`.
```
Comment (jieru-hu) [inline website/versioned_docs/version-1.0/tutorials/basic/running_your_app/6_tab_completion.md:18]:
```suggestion
Get the exact command to install the completion from `python <your-hydra-app> --hydra-help`.
```

**Comment by jieru-hu** [comment]
Thanks! One nit, otherwise lgtm.

**Comment by elisim** [comment]
Great

**Comment by jieru-hu** [comment]
awesome, thanks!

**Pull request merged**